eng/scripts/Language-Settings.ps1

```diff
@@ -203,25 +203,27 @@ function DockerValidation{
     [Parameter(Mandatory=$false)]
     [string]$workingDirectory
   ) 
+  $commandLine = ""
   if ($PackageSourceOverride) {
-    docker run -v "${workingDirectory}:/workdir/out" -e TARGET_PACKAGE=$packageName -e TARGET_VERSION=$packageVersion `
-       -e EXTRA_INDEX_URL=$PackageSourceOverride -t $DocValidationImageId 2>&1 | Out-Null
+    $commandLine = docker run -v "${workingDirectory}:/workdir/out" -e TARGET_PACKAGE=$packageName -e TARGET_VERSION=$packageVersion `
+       -e EXTRA_INDEX_URL=$PackageSourceOverride -t $DocValidationImageId 2>&1
   }
   else {
-    docker run -v "${workingDirectory}:/workdir/out" `
+    $commandLine = docker run -v "${workingDirectory}:/workdir/out" `
       -e TARGET_PACKAGE=$packageName -e TARGET_VERSION=$packageVersion -t $DocValidationImageId 2>&1 | Out-Null
```

Comment:
Did you want to remove the Out-Null here as well?

Reply:
No. That's for testing purpose. 
I am kicking a pipeline.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1276768&view=results